juntagrico/dao/jobdao.py

```diff
@@ -77,4 +77,5 @@ def get_promoted_jobs():
 
     @staticmethod
     def upcomming_jobs_for_member(member):
-        return Job.objects.filter(time__gte=timezone.now(), assignment__member=member).distinct()
+        upcoming_jobs = Job.objects.filter(time__gte=timezone.now(), assignment__member=member).order_by('time')
```

Comment:
The .distinct() is still needed otherwise a job will be shown multiple times if the member signed up for it with more than one person.

Reply:
Thanks for catching that. I was wondering about the `distinct` and eventually concluded that signing up with more than one person was handled by `Assignment.amount`. So what's `amount` for then?

Reply:
@pascalfree ?

Reply:
The amount is for jobs/assignment that count more or less than 1. It's set according to the job multiplier on signing up.
It's not related to how many slots the assignment occupies.